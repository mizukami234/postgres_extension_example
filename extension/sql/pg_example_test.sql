CREATE EXTENSION IF NOT EXISTS "pg_example";
CREATE TABLE t(i int);
INSERT INTO t VALUES (1), (3);
SELECT * FROM t;
TRUNCATE t;
SELECT * FROM t;
DROP TABLE t;