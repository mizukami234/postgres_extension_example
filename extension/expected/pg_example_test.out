CREATE EXTENSION IF NOT EXISTS "pg_example";
CREATE TABLE t(i int);
INSERT INTO t VALUES (1), (3);
SELECT * FROM t;
 i
---
 1
 3
(2 rows)

TRUNCATE t;
ERROR:  do not use truncate
SELECT * FROM t;
 i 
---
 1
 3
(2 rows)

DROP TABLE t;
